contracts/Safe.sol

```diff
@@ -318,7 +318,7 @@ contract Safe is
         return keccak256(abi.encode(DOMAIN_SEPARATOR_TYPEHASH, getChainId(), this));
     }
 
-    /// @dev Returns the bytes that are hashed to be signed by owners.
+    /// @dev returns the pre-image of the transaction hash (see getTransactionHash)
```

Comment:
Why do we lower case this? For all other methods this is upper case and a sentence (ends with a `.`)

Reply:
this is pure gen z writing. will revert that change